azerothcore/azerothcore-wotlk
# fix(Core\Spells):Food from Bobbing Apple should give Well Fed after 10s not when breaking the eating

## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8659
- Closes https://github.com/chromiecraft/chromiecraft/issues/2160

## SOURCE:


## Tests Performed:

- .add 20516 and use it, notice you won't get the Well Fed after 10s but when you break the eating effect since it has entry in spell_linked_spell
- 


## How to Test the Changes:


1. .add 20516 and use it, notice you will gain Well Fed after 10s of eating and not when breaking the effect
2.
3.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
